buttons/imagefile.go

```diff
@@ -19,9 +19,9 @@ type ImageFileButton struct {
 }
 
 // GetImageForButton is the interface implemention to get the button's image as an image.Image
-func (btn *ImageFileButton) GetImageForButton() image.Image {
+func (btn *ImageFileButton) GetImageForButton(btnSize int) image.Image {
 	// TODO base the 96 on the image bounds
-	newimg := image.NewRGBA(image.Rect(0, 0, 96, 96))
+	newimg := image.NewRGBA(image.Rect(0, 0, btnSize, btnSize))
```

Comment:
More updates that replace `ButtonSize` with `btnSize`. This is all over the PR.

In almost all cases it may be worth it to look into seeing if it's viable to add the size field to a type.